movement.h

```diff
@@ -0,0 +1,306 @@
+class movements{
+    public:
+    int motores[8];
+    int pwm[4];
+
+    movements(int m1, int m2, int m3, int m4, int m5, int m6, int m7, int m8, int p1, int p2, int p3, int p4);
+    void movement(int direccion);
+    void angleMovement(int angle);
+    void pwmNormal();
+    void pwmAtras();
+    void pwmCustom(float f1, float f2, float f3, float f4);
+    void seguirPelota();
+};
+
+movements::movements(int m1, int m2, int m3, int m4, int m5, int m6, int m7, int m8, int p1, int p2, int p3, int p4){
+    motores[0] = m1;
+    motores[1] = m2;
+    motores[2] = m3;
+    motores[3] = m4;
+    motores[4] = m5;
+    motores[5] = m6;
+    motores[6] = m7;
+    motores[7] = m8;
+    pwm[0] = p1;
+    pwm[1] = p2;
+    pwm[2] = p3;
+    pwm[3] = p4;
+}
+
+void movements::movement(int direccion){
+    /* A switch case is used to determine one of the 9 possible movements that the robot can perform.
+    *  An array is used to determine the pins of the motors and the pwms. 
+    *  The first two spaces on the 'motores' array should be used for the front left wheel,
+    *  the next two for the front right wheel, the next two for the back right wheel and the last two fote the back left wheel. 
+    */
+  switch(direccion){
+
+    case 1: //FRONT
+      pwmNormal();
+      for(int i=0; i<8; i++){

```

Comment:
Think of a better way to assign directions. One more explicit to say which motor is going forward,backward or stop.
Maybe something like this:
`
void Movement::setDirection(const int angle) {
  switch(angle) {
    case 0:
      move0();
    break;
    ....
  }
}
`
`
void Movement::move0() {
  front_left_motor_.forward();
  back_left_motor_.backward();
  front_right_motor_.backward();
  back_right_motor_.forward();
}
`